docs/development/issues.rst

```diff
@@ -8,5 +8,5 @@ functionalities are found, file an issue on our official `GitHub
 <https://github.com/tardis-sn/tardis>`_ page.
 
 We encourage any interested user to actively contribute to the development of
-TARDIS. Just follow some basic principles outlined on our :doc:`Contributing
-HowTo <workflow/development_workflow>` and get started.
+TARDIS. Just follow some basic principles outlined on our :ref:`Contributing
+HowTo <development-workflow>` and get started.
```

Comment:
I can't find any document which has `ref: development-workflow`. Is there any?

Also instead of saying it "Contributing HowTo", rename it to actual section you intend to cross-reference.

Reply:
The section is called "General Workflow to add a new feature" and I don't think that would read well there. I can change it, but it just doesn't sound right to me.